src/vendor/vendor.c

```diff
@@ -66,22 +66,27 @@
 #include <ccsp_base_api.h>
 #include <dslh_definitions_database.h>
 
+#include <rbus.h>
```

Comment:
here are the fundamental checkpoint i just want to check
1.Is DAC been tested using usppa cli (installdu/unstall/exection start/stop)
2.Datamodel individual params get an set from controller
3.Device. using usppa-cli and from controller
4.Is commands been working

Reply:
Can you please share how to test DAC using usppa cli (installdu/unstall/exection start/stop)
Will attach the get /set from controller as well as usppa-cli results in ticket